tests/Sniffs/Functions/data/MultiLineFunctionDeclarationSniff.OK.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+class Test
+{
+	public function __construct(private string $str){}
```

Comment:
I would suggest we should add a rule that forces the opening bracket to have exactly one space between itself and the previous non whitespace token?

Reply:
I agree. I would say that only D and E are "OK" in this file.